### ISSUE TITLE
Confusing error shown when adding new DB fails: "Success" and "The connection attempt failed."

### ISSUE THREAD

**Issue by max-rocket-internet** [metabase/metabase#12973]
**Describe the bug**

Conflicting messages with both `Success` and `The connection attempt failed.` shown:

**Logs**

Not required.

**To Reproduce**

Add a new database in the UI with a trailing space, like `my-app.cluster-ro-xxxxxxxxxx.eu-west-1.rds.amazonaws.com`

**Expected behavior**

Don't show a green `Success` button.

**Screenshots**

![Screen Shot 2020-07-22 at 10 01 45](https://user-images.githubusercontent.com/8859277/88152172-f411d900-cc03-11ea-8c85-b7bf8eec045d.png)

**Information about your Metabase Installation:**

```json
{
  "browser-info": {
    "language": "en-GB",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.5+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.5",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.5+10",
    "os.name": "Linux",
    "os.version": "4.14.177-139.253.amzn2.x86_64",
    "user.language": "en",
    "user.timezone": "UTC"
  },
  "metabase-info": {
    "databases": [
      "postgres",
      "mysql"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "9.6.16"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-04-02",
      "tag": "v0.35.1",
      "branch": "release-0.35.x",
      "hash": "e67f169"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Severity**

Minor

**Comment by flamber** [comment]
Hi @max-rocket-internet 
That was fixed in on of the minor versions of 0.35.x - latest release in that series is 0.35.4 - latest release is 0.36.0 (yesterday)

**Comment by max-rocket-internet** [comment]
OK great 🙂
